docs/anomaly-detection/QuickStart.md

```diff
@@ -16,18 +16,19 @@ aliases:
 For service introduction visit [README](/anomaly-detection/) page
 and [Overview](/anomaly-detection/overview.html) of how `vmanomaly` works.
 
-## How to install and run `vmanomaly`
+## How to install and run vmanomaly
```

Comment:
We need to mark that it's a keyword/name. If `backticks` arent't good, try Capitalize.

Also there are several occurences of backticked `vmanomaly` below, let's be consistent.

Reply:
agree, I prefer to use `vmanomaly` I just returned it back in headers to a simple text, as there was broken rendering:
![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/11490990/ec78bdf1-ed0f-4936-a58d-7f35ae06e4a6)